crates/re_arrow_store/src/store_event.rs

```diff
@@ -234,11 +236,11 @@ mod tests {
     };
     use re_types_core::{components::InstanceKey, Loggable as _};
 
-    use crate::{DataStore, GarbageCollectionOptions};
+    use crate::{DataStore, GarbageCollectionOptions, StoreView, StoreViewHandle};
 
     use super::*;
 
-    /// A simple store view for test purposes that keeps track of the quantity of data available
+    /// A simple [`StoreView`] for test purposes that keeps track of the quantity of data available
```

Comment:
I guess that's one of the next PRs leaking in, but the thing I'm reading right now isn't a `impl StoreView` and also doesn't need those imports ;)